### PR TITLE
Space Travelers' Hub

### PR DESCRIPTION
**Pull request by MostafaShahat2021**
## In this PR:
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that we used the correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
 - Make sure that we documented our work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
 - Follow Microverse's list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

- ⭐️Install [React Redux](https://react-redux.js.org/), [Redux Logger](https://www.npmjs.com/package/redux-logger) and [React Router](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/react_router.md).
-⭐️Download the [free image](https://www.flaticon.com/free-icon/planet_3212567?term=space&page=1&position=19&page=1&position=19&related_id=3212567&origin=style) for the app logo.
-⭐️Create routes and view components: rockets, missions, my profile. Add dragons [only if your team has 3 members]).
- ⭐️ Use <NavLink /> for the page navigation links and style active class to indicate which section/page user is currently on (underline active navigation link).
- ⭐️Create directories for all Redux state slice files: rockets, missions.
- ⭐️Upon first render fetch data from the SpaceX API endpoints:
    - Rockets: https://api.spacexdata.com/v4/rockets.
    - Missions: https://api.spacexdata.com/v3/missions.
- ⭐️Use ```useSelector()``` Redux Hook to select the state slices and render lists of rockets and missions in corresponding routes.
- ⭐️Add actions and reducers for booking rockets and joining missions.
- ⭐️Add actions and reducers for canceling rockets/dragons and leaving missions.
- ⭐️Add conditional components rendering.
- ⭐️Add  My Profile page.
- ⭐️Set up testing libraries(React Testing Library and Jest).
- ⭐️Add test for all components.

### CODE REVIEW

File: src/Pages/rockets/Rockets.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  width: 70%;
+}
+
+.description {
+  line-height: 2em;
+  margin: 1rem;
+}
```
Comment (OyePriscilla) [inline src/Pages/rockets/Rockets.css:24]: - [ ] Kindly arrange these items in accordance to the given template, reproducing a template perfectly shows how well we give attention to details. ℹ️ 

## Your Design
Untitled

## Given template
Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @MostafaShahat2021 and @NalianAchei2020 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Used React components, props and Router ✔️ 
Connected React and Redux ✔️ 
Handled events in a React app ✔️ 
Write unit tests with React Testing Library ✔️ 
Use store, actions and reducers in React ✔️ 
Performed a code review for a team member ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MostafaShahat2021** [comment]
Hi @OyePriscilla Thanks a lot 💯 for your efforts in reviewing our code 😄 .

**Review by skyv26 — APPROVED** [review]
## Hi @MostafaShahat2021 and @NalianAchei2020 🌹 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well-structured project ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MostafaShahat2021** [comment]
Hi 👋, @skyv26 Thanks a lot :100:  for your efforts in reviewing our code, we really appreciate it 😊.

**Pull request merged**